playground/frontend/playground_components/assets/symbols/python.g.yaml

```diff
@@ -4790,6 +4790,10 @@ GBKTransform:
   - from_runner_api_parameter
   - to_runner_api_parameter
 GcpTestIOError: {}
+GcsDownloader:
```

Comment:
we should also revert this line: https://github.com/apache/beam/blob/master/CHANGES.md#ios

Python GCSIO is now implemented with GCP GCS Client instead of apitools (https://github.com/apache/beam/issues/25676)

Reply:
Done